svc-bgs-api/src/docker/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM ruby:2.7.7-alpine3.16
- 
+#FROM ruby:2.7.7-alpine3.16
+FROM ruby:2.7.7
```

Comment:
Ran into a [build error](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/4629816819) and had to make this change and comment out `apk` commands below. 
@greene858, this blocks deployment and testing unless you resolve SecRel alerts from not using the `alpine` image.

Reply:
I revised the app and Dockerfile to address this